setup.py

```diff
@@ -29,14 +29,16 @@
     long_description=long_description,
     install_requires=[
         "requests >= 2.0.1",
+        "six"
     ],
-	classifiers=[
-		'Programming Language :: Python',
-		'Development Status :: 3 - Alpha',
-		'Environment :: Web Environment',
+    classifiers=[
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3.4',
```

Comment:
I wonder whether we need to provide two versions for pip now, one for Python 2 and one for Python 3